tests/shuttleSchedule.test.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import ShuttleSchedule from '../components/home/menu/shuttleBusSchedule';
+
+beforeEach(() => {
+
+});
+
+it('Should set index of the button to 0', () => {
+  const shuttleScheduleComponent = renderer.create(<ShuttleSchedule />).getInstance();
+  shuttleScheduleComponent.updateButtonIndex(0);
+  expect(shuttleScheduleComponent.state.selectedButtonIndex).toBe(0);
+});
+
+it('Should set index of the button to 1', () => {
+  const shuttleScheduleComponent = renderer.create(<ShuttleSchedule />).getInstance();
+  shuttleScheduleComponent.updateButtonIndex(1);
+  expect(shuttleScheduleComponent.state.selectedButtonIndex).toBe(1);
+});
+
+it('Should return times for Loyola\'s shuttle service', () => {
+  const shuttleScheduleComponent = renderer.create(<ShuttleSchedule />).getInstance();
+  const response = shuttleScheduleComponent.getShuttleCampusInformation(1);
+  expect(response).toStrictEqual(
```

Comment:
maybe instead of putting the data directly into the assert, put it in a var when invoking beforeEach(). This way, the assert can be more readable via expect(response).toStrictEqual(loyolaScheduleMonThu)